src/console/writer.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Writes formatted arguments to console;
+ */
+
+export default class ConsoleWriter {
+    constructor () {
+        this._console = console ? console : {};
+    }
+
+    /**
+     * @param {String} method
+     * @param {String} prefix
+     * @param {Array<*>} formattedArgs
+     */
+    write (method, prefix, formattedArgs) {
+        if (this._console[method]) {
```

Comment:
will not work in IE. We should explicitly call console.error(), console.info(), etc